generated-dockerfiles/rapidsai_centos7-base.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# RAPIDS Dockerfile for centos7 "base" image
+#
+# base: RAPIDS is installed from published conda packages to the 'rapids' conda
+# environment.
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+ARG CUDA_VER=10.1
+ARG LINUX_VER=centos7
+ARG PYTHON_VER=3.7
+ARG RAPIDS_VER=0.17
+ARG FROM_IMAGE=rapidsai/rapidsai-core
```

Comment:
Need to check in about naming these images.

Reply:
this is correct